src/seed_data/seed.ts

```diff
@@ -0,0 +1,67 @@
+import { Connection, createConnection, getConnection, getConnectionOptions } from "typeorm";
+import { runSeeder, useSeeding } from "typeorm-seeding";
+import { Course } from "../entity/Course";
+import { Exam } from "../entity/Exam";
+import { Note } from "../entity/Note";
+import { Student } from "../entity/Student";
+import { StudentCourse } from "../entity/StudentCourse";
+import { StudentExam } from "../entity/StudentExam";
+import CreateCourses from "./seeds/createCourses.seed";
+import CreateStudents from "./seeds/createStudents.seed";
+
+import * as ora from 'ora';
+
+const seed = async () => {
+    const connectionOptions = await getConnectionOptions('dev');
+    createConnection({ ...connectionOptions, name: "default" }).then(async () => {
+        const spinner = ora();
+        spinner.start('Setting up seeding utilities');
+
+
+        const connection: Connection = getConnection("default");
+        await useSeeding({connection: connectionOptions.name})
+            .then((_) => spinner.succeed())
+            .catch(error => {
+                console.log(error);
+                spinner.fail();
+                return;
+            });
+
+        // This deletes all data, regardless of foreign keys. Should never happen in production.
```

Comment:
looks like it's clearing each table before creating data, so i was thinking if it's worth to create a separate database connection in `ormconfig` with the `dropSchema: true` attached and then we wouldn't need to worry about losing local data saved.
`dropSchema` enabled would clear the database for you on every new connection.
 
what do you think?

Reply:
Documenting the discussion from today's meeting: We decided rather to go with migrations, which will be set up in #68, and have the run seeder be the first migration we run. This will keep the amount of time spent on maintaining the factories and seeders low, and will establish a good-practice workflow for our team in terms of how often we actually seed the DB.